tests/conftest.py

```diff
@@ -11,7 +11,11 @@
 
 @pytest.fixture
 def aau_token_path():
-    return join(tokens.CONFIG_DIR, "aau_token")
+    old_dir = tokens.CONFIG_DIR
```

Comment:
Since we're manipulating the client token so much in testing now I want to make sure to give the code a temporary test location for storage

Reply:
oops I missed this sorry, will get it done